aten/src/ATen/native/Convolution.cpp

```diff
@@ -500,13 +500,18 @@ struct ConvParams {
     if (!at::globalContext().userEnabledMkldnn()) {
       return false;
     }
+    if (transposed && is_output_padding_big()) {
+      return false;
+    }
+    if (transposed && groups > 1 && at::symint::size<T>(input, 1) == groups) {
+      return false;
+    }
```

Comment:
Add comments on the reason why we want fallback here?

Reply:
bug in `ideep` that recognizes a 4d tensor with groups (and C=1) to be a channels_last_3d tensor. in oneDNN, g is treated as a separate dimension.

will have it fixed.